src/styles/tutorials.scss

```
@@ -1,52 +1,52 @@
 .driver-disabled {
-    border: 1.5px solid #ddd !important;
+  border: 1.5px solid #ddd !important;
 }
 
 .driver-btn-group button {
-    font-family: 'Nunito', sans-serif !important;
-    font-size: 14px !important;
-    border: 1.5px solid #42b983 !important;
-    text-shadow: none !important;
-    border-radius: 0px !important;
+  font-family: "Nunito", sans-serif !important;
```

Stylelint (17.11.0)

[error] Expected no quotes around "Nunito" (font-family-name-quotes)

(font-family-name-quotes)